project_resource_calendar/wizard/weekly_report_wizard.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime, timedelta
+
+from odoo import models, fields, api
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT as DATE_FORMAT
+
+
+class WeeklypReportWizard(models.TransientModel):
+    _name = 'weekly.report.wizard'
+
+    date_start = fields.Date(string="Start Date", required=True, default=fields.Date.today)
+    date_end = fields.Date(string="End Date", required=True, default=fields.Date.today)
```

Comment:
@eilst as an enhancement: can we have by default the date start is the current monday and the end date is current friday?

Reply:
done